deploy/helm/charts/values.yaml

```diff
@@ -0,0 +1,135 @@
+# - name: "image-pull-secret"
+
+zfsNode:
+  componentName: openebs-zfs-node
+  driverRegistrar:
+    name: "csi-node-driver-registrar"
+    image:
+      # Make sure that registry name end with a '/'.
+      # For example : quay.io/ is a correct value here and quay.io is incorrect
+      registry: quay.io/
+      repository: k8scsi/csi-node-driver-registrar
+      pullPolicy: IfNotPresent
+      # Overrides the image tag whose default is the chart appVersion.
+      tag: v1.2.0
+  updateStrategy:
+    type: RollingUpdate
+  annotations: {}
+  podAnnotations: {}
+  resources: {}
+  # This can be configured to run on various different k8s ditributions like
+  # microk8s where kubelet dir is different
+  kubeletDir: "/var/lib/kubelet/"
+  # limits:
+  #   cpu: 10m
+  #   memory: 32Mi
+  # requests:
+  #   cpu: 10m
+  #   memory: 32Mi
+  ## Labels to be added to openebs-zfs node pods
+  podLabels:
+    name: openebs-zfs-node
+  nodeSelector: {}
+  tolerations: []
+  securityContext: {}
+  labels: {}
+
+controller:
+  componentName: openebs-zfs-controller
+  resizer:
+    name: "csi-resizer"
+    image:
+      # Make sure that registry name end with a '/'.
+      # For example : quay.io/ is a correct value here and quay.io is incorrect
+      registry: quay.io/
+      repository: k8scsi/csi-resizer
+      pullPolicy: IfNotPresent
+      # Overrides the image tag whose default is the chart appVersion.
+      tag: v0.4.0
+  snapshotter:
+    name: "csi-snapshotter"
+    image:
+      # Make sure that registry name end with a '/'.
+      # For example : quay.io/ is a correct value here and quay.io is incorrect
+      registry: quay.io/
+      repository: k8scsi/csi-snapshotter
+      pullPolicy: IfNotPresent
+      # Overrides the image tag whose default is the chart appVersion.
+      tag: v2.0.1
+  snapshotController:
+    name: "snapshot-controller"
+    image:
+      # Make sure that registry name end with a '/'.
+      # For example : quay.io/ is a correct value here and quay.io is incorrect
+      registry: quay.io/
+      repository: k8scsi/snapshot-controller
+      pullPolicy: IfNotPresent
+      # Overrides the image tag whose default is the chart appVersion.
+      tag: v2.0.1
+  provisioner:
+    name: "csi-provisioner"
+    image:
+      # Make sure that registry name end with a '/'.
+      # For example : quay.io/ is a correct value here and quay.io is incorrect
+      registry: quay.io/
+      repository: k8scsi/csi-provisioner
+      pullPolicy: IfNotPresent
+      # Overrides the image tag whose default is the chart appVersion.
+      tag: v1.6.0
+  updateStrategy:
+    type: RollingUpdate
+  annotations: {}
+  podAnnotations: {}
+  resources: {}
+  # limits:
+  #   cpu: 10m
+  #   memory: 32Mi
+  # requests:
+  #   cpu: 10m
+  #   memory: 32Mi
+  ## Labels to be added to openebs-zfs controller pods
+  podLabels:
+    name: openebs-zfs-controller
+  nodeSelector: {}
+  tolerations: []
+  securityContext: {}
+
+zfsPlugin:
```

Comment:
What is this for? csi plugin?

Reply:
Yes , its a common tag used in controller and node files